tests/Aspire.Hosting.Redis.Tests/XunitAttributes.cs

```diff
@@ -0,0 +1,6 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+[assembly: CollectionBehavior(DisableTestParallelization = true)]
```

Comment:
NIT: This should be a shared file referenced by all projects as opposed to a file in each test project.

Reply:
Done